scripts/clearml_agent/mount_aqua_on_data.bat

```diff
@@ -0,0 +1,9 @@
+rem Download rclone from: https://rclone.org/
+rem Extract into a folder.
+rem Copy this file and rclone.conf to the new folder
+rem You will also need the api key and secret installed through awscli for this to work.
+rem Note that you can map \\wsl$\Ubuntu to a drive letter and then browse the data.
+call wsl.exe fusermount -uz /data/aqua-ml-data
```

Comment:
What about Linux? Also, maybe this path should be specified by the `SIL_NLP_DATA_PATH` environment variable.